### ISSUE TITLE
'Showing rows 0 to 11 of 11' for paginationShowsTotal should start at 1 instead of 0

### ISSUE THREAD

**Issue by jflayhart** [AllenFang/react-bootstrap-table#668]
This to me is not professional to the end user. It should say `Showing rows 1 to 11 of 11` not use 0 as a start number... Or is there a way currently to change that?

**Comment by jflayhart** [comment]
And I wonder if it should be "items" or "entries" instead of rows. If I have time I could probably work on this PR, but wanted to see what every one else thinks.

**Comment by AllenFang** [comment]
@jflayhart, ```paginationShowsTotal``` accept a bool or function, if a function given, you should return a string, check following example:

```js
renderShowsTotal(start, to, total) {
    return (
      <p style={ { color: 'blue' } }>
        From { start +1 } to { to + 1 }, totals is { total }&nbsp;&nbsp;(its a customize text)
      </p>
    );
}
const options = {
      paginationShowsTotal: this.renderShowsTotal  // Accept bool or function
};
return (
//....
);
```

I think you are right, maybe I can make this enhancement in next version or ```v3.0.0```. Thanks :)

**Comment by AllenFang** [comment]
hi @jflayhart, this issue fixed on ```v2.5.1```. Thanks :)
